src/components/UsernameForm/CompletionMessage.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const CompletionMessage = () => {
+  return (
+    <div className="text-center text-white pb-2" style={completionMessageStyle}>
+      <p className="text-2xl">Thank you for your contributions!</p>
+      <p className="text-xl">You have sucessfully completed Frogtoberfest 2019</p>
+    </div>
+  );
+};
+
+const completionMessageStyle = {
```

Comment:
You can use `text-green` class instead of this here. Avoid `inline` styling as much as possible. We can use tailwind css utility classes.

Reply:
Also,

```diff
- sucessfully
+ successfully
```

Reply:
Add a `!` at the end.